app/src/pages/playground/Playground.tsx

```diff
@@ -254,35 +257,37 @@ function PlaygroundContent() {
       >
         <Panel>
           <div css={playgroundPromptPanelContentCSS}>
-            <Accordion arrowPosition="start" size="L">
-              <AccordionItem
-                title="Prompts"
-                id="prompts"
-                extra={
-                  <Flex direction="row" gap="size-100" alignItems="center">
-                    <TemplateLanguageRadioGroup />
-                    <AddPromptButton />
-                  </Flex>
-                }
-              >
-                <div css={promptsWrapCSS}>
-                  <Flex direction="row" gap="size-200" maxWidth="100%">
-                    {instances.map((instance) => (
-                      <View
-                        flex="1 1 0px"
-                        key={instance.id}
-                        minWidth={PLAYGROUND_PROMPT_PANEL_MIN_WIDTH}
-                      >
-                        <PlaygroundTemplate
+            <DisclosureGroup defaultExpandedKeys={["prompts"]}>
+              <Disclosure id="prompts">
```

Comment:
This originally had a size of "L". Stashing and unstashing this, I can't really tell a difference in size

Reply:
I think you may be right that there's only one size

Reply:
Turns out L was valuable. It made the accordions on this page all the same height, which matters when one accordion has nested controls